src/main/archetype/ui.tests/test-module/specs/aem/forms.js

```diff
@@ -0,0 +1,153 @@
+
+                //First open structure layer
+                expect($(selectors.editor.layerSwitcher).waitForDisplayed()).toBe(true);
+                $(selectors.editor.layerSwitcher).click();
+                expect($(selectors.editor.layerSelector.structure).waitForDisplayed()).toBe(true);
+                $(selectors.editor.layerSelector.structure).click();
+
+                //Verify all three containers are visible
+                expect($(getContainerSelector(templateStructure,'parsys1'))).toBeDisplayed();
+                expect($(getContainerSelector(templateStructure,'parsys2'))).toBeDisplayed();
+                expect($(getContainerSelector(templateStructure,'guideContainer'))).toBeDisplayed();
+
+                //Verify Applied Policy Content
+                $(getContainerSelector(templateStructure,'guideContainer')).click();
+                expect($(selectors.editor.templateEditor.structureLayer.policyButton).waitForDisplayed()).toBe(true);
+                $(selectors.editor.templateEditor.structureLayer.policyButton).click();
+                expect($(selectors.editor.templateEditor.structureLayer.policyPage.allowedComponent.adaptiveFormGroup + '[checked]').waitForDisplayed()).toBe(true);
+                expect($(selectors.editor.templateEditor.structureLayer.policyPage.cancel).waitForDisplayed()).toBe(true);
+                $(selectors.editor.templateEditor.structureLayer.policyPage.cancel).click();
+
+                //Switch to initial content layer
+                expect($(selectors.editor.layerSwitcher).waitForDisplayed()).toBe(true);
+                $(selectors.editor.layerSwitcher).click();
+                expect($(selectors.editor.layerSelector.initial).waitForDisplayed()).toBe(true);
+                $(selectors.editor.layerSelector.initial).click();
+                expect($(afContainerOverlaySelector).waitForDisplayed()).toBe(true);
+
+                //Verify no console error present in structure and initial content layer
+                expect(consoleErrors).toHaveLength(0);
+
+                //Open Side panel, verify that no invalid item in properties tab and then close the side panel
+                expect($(selectors.editor.sidePanelToggleButton).waitForDisplayed()).toBe(true);
+                $(selectors.editor.sidePanelToggleButton).click();
+                expect($(selectors.editor.sidePanelOpen).waitForDisplayed()).toBe(true);
+                expect($(selectors.editor.sidePanel.propertiesTab + '.is-invalid').waitForDisplayed({reverse : true})).toBe(true);
+                $(selectors.editor.sidePanelToggleButton).click();
+                expect($(selectors.editor.sidePanelOpen).waitForDisplayed({reverse : true})).toBe(true);
+
+                //Preview the template using url and verify AF field presence in the form
+                browser.url(`${config.aem.author.base_url}/${templatePath}/initial.html?wcmmode=preview`);
```

Comment:
Just preview and check for no console errors

Reply:
Now previewing, checking AF field presence and console log verification only.